### ISSUE TITLE
[FORMATTING]  `mysql` keyword conflicts with the default `mysql` schema  

### ISSUE THREAD

**Issue by duongthienlee** [sql-formatter-org/sql-formatter#629]
**Input data**
If we use `upper` as the value for `keywordCase`, the following query will not work
```sql
select * from mysql.user
```

**Expected Output**

```sql
SELECT
  *
FROM
  mysql.user
```

**Actual Output**

```sql
SELECT
  *
FROM
  MYSQL.user
```

**Usage**

- How are you calling / using the library?
```
import { formatDialect, mariadb } from 'sql-formatter'
export function formatSQL(v) {
    return formatDialect(v, { dialect: mariadb, tabWidth: 2, keywordCase: 'upper' })
}
```
- What SQL language(s) does this apply to? MariaDB
- Which SQL Formatter version are you using? 12.2.3

Another case:
`select * from user` is formatted to
 ```
SELECT
  *
FROM
  USER
 ```
 
I think the identifiers shouldn't be transformed to uppercase. Btw, I think if we decide to use keywords as reserved words, then
https://mariadb.com/kb/en/reserved-words/ should be used instead of this one https://mariadb.com/kb/en/information-schema-keywords-table/

**Comment by nene** [comment]
Thanks for reporting. This definitely needs improvement.

There's currently two bad options for these keyword lists:

- by including only the reserved keywords, we avoid the problem of accidentally uppercasing ordinary identifiers, but failing to uppercase some keywords.
- by including all the keywords, we make sure to uppercase them all, but run into the trouble of uppercasing ordinary identifiers (like seen here).

The first option is definitely the safer way to go. In some SQL dialects the actual reserved words list is fairly short, and I've hard to include also a large amount of the unreserved keywords, but in case of MySQL & MariaDB there's a load of reserved keywords which really should cover the majority of cases.

PS. Thanks for the links to docs.

**Comment by nene** [comment]
Trimmed down the list of keywords. Fixed in latest master. Hopefully soon in the next release.

**Comment by nene** [comment]
Published the fix in 13.0.0
